tests/braille-specs/lg-ug-g1.yaml

```diff
@@ -0,0 +1,4185 @@
+# Luganda (Uganda) Braille Grade 1
+# Copyright (C) 2022 by Compass Braille <http://www.compassbraille.org>
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved. This file is offered as-is,
+# without any warranty.
+#
+# ----------------------------------------------------------------------------------------------
+display: |
+  include unicode.dis
+  display 9 9
+  display a a
+table:
+  language: lg
+  region: ug
+  grade: 1
+  __assert-match: lg-ug-g1.utb
+
+flags: {testmode: bothDirections}
+tests:
+  - ['\x0009', '9']
+  - ['\x0020', '\x2800']
+  - ['\x00a0', 'a']
+flags: {testmode: forward}
+tests:
+  - ['\x000a', '\x2800']
+  - ['\x000b', '\x2800']
+  - ['\x000c', '\x2800']
+  - ['\x000d', '\x2800']
+  - ['\x2000', '\x2800']
+  - ['\x2001', '\x2800']
+  - ['\x2002', '\x2800']
+  - ['\x2003', '\x2800']
+  - ['\x2004', '\x2800']
+  - ['\x2005', '\x2800']
+  - ['\x2006', '\x2800']
+  - ['\x2007', '\x2800']
+  - ['\x2008', '\x2800']
+  - ['\x2009', '\x2800']
+  - ['\x200a', '\x2800']
+  - ['\x2028', '\x2800']
+  - ['\x2029', '\x2800']
+  - ['\x202f', '\x2800']
+  - ['\x205f', '\x2800']
+
+# Work forward only:
+flags: {testmode: forward}
+tests:
+
+# Included UEB table backtranslates both single and double directional quotes
+# as generic non-directional double quotes.
+  - ['“Tugende', '⠦⠠⠞⠥⠛⠑⠝⠙⠑'] # backtranslated as: "Tugende
+  - ['bangi.”', '⠃⠁⠝⠛⠊⠲⠴'] # backtranslated as: bangi."
+  - ['sabakusaani?”', '⠎⠁⠃⠁⠅⠥⠎⠁⠁⠝⠊⠦⠴'] # backtranslated as: sabakusaani?"
+  - ['gonna?”', '⠛⠕⠝⠝⠁⠦⠴'] # backtranslated as: gonna?"
+  - ['‘Mukola', '⠠⠦⠠⠍⠥⠅⠕⠇⠁'] # backtranslated as: "Mukola
+  - ['gonna?’', '⠛⠕⠝⠝⠁⠦⠠⠴'] # backtranslated as: gonna?"
+  - ['“', '⠦'] # backtranslated as: "
+  - ['‘', '⠠⠦'] # backtranslated as: "
+  - ['”', '⠴'] # backtranslated as: "
+  - ['’', '⠠⠴'] # backtranslated as: "
+
+
+###############################################################
+# List of words which work in both directions:
```

Comment:
Were these words actually proofread or is the list auto-generated? A comment would be good.

Reply:
The braille book was proof read using the current table then I autogenerated the tests by generating a list of unique words (well strings) from the book and using the same table on them individually.
I hope the comment in the tests makes this clear.

Reply:
Thanks!